web/client/components/misc/enhancers/localizedProps.js

```diff
@@ -33,12 +42,12 @@ const accumulate = (props, messages) => (acc = {}, propName) => ({
  * //...
  * <Input placeholder="path.to.placeholder.message" />
  */
-module.exports = (propNames) => compose(
+module.exports = (propNames, labelName) => compose(
```

Comment:
- call this localizedKey
- document the new prop and how to use in case of array